examples/web-color/README.md

```diff
@@ -0,0 +1,136 @@
+# Web color example
+
+This example shows the mechanics of applying patches to a register. The data
+is the list of web colors as defined by the [W3C](https://www.w3c.org), in
+particular [HTML4](https://www.w3.org/TR/1999/REC-html401-19991224/) and
+[CSS Color Module Level 3](https://www.w3.org/TR/css-color-3/).
+
+
+## Setup
+
+Ensure that you have Python +3.6 installed on your machine.
+
+Then, install the `registers` command line interface:
+
+```sh
+pip3 install --user "git+https://github.com/openregister/registers-cli#egg=registers"
```

Comment:
This command didn't work for me due to conflicting installed dependencies. Perhaps you can include a virtualenv in the install to make the command more robust?

Reply:
This command does not make `registers` command available for me, are you sure there is no need to add something to `PATH`?

Reply:
If you haven't set `PATH` for user python installs pip mentions that, or at least it did for me. I don't think a tutorial should double on these but I'll think about it.

Regarding the virtualenv, I was desperately trying to avoid that. If you need to setup a virtualenv it should be your choice, not the tutorial's suggestion.

Reply:
I added a troubleshooting section. That said, both issues you mention are not related to this example nor to this project. Are very common situations in Python/Pip.